### DISCUSSION TITLE
AF/IF Noise Filters?

### DISCUSSION THREAD

**Discussion by sefirot43** [AlexandreRouma/SDRPlusPlus#183] [Ideas]
I am following and using closely all the updates of this promising software, are AF/IF noise filters been considered? Thank you for your hard work.

**Comment by AlexandreRouma** [comment]
AF noise filters are on the today list but not IF

**Comment by Jempiken** [comment]
Meaning a Noise Blanker...?
Some Fence peaks are disturbing signals on HF Bands
Tnx Alex

**Comment by sefirot43** [comment]
Hello, still noise reduction is needed to complete the release version, nothing more.

**Comment by sefirot43** [comment]
Good morning Alexandre, enjoying your program!
Any news on the quote of pbij2001 on this thread, with regard to the windows environment?
Thank you.

**Reply by AlexandreRouma** [reply]
I've been too busy with university to do any kind of serious DSP work like this unfortunately

**Comment by om3bc** [comment]
Good morning Alexandre, enjoying your program!
I tried to receive fast data (GMSK, GFSK 9600dB) from satellites - without success. How can I turn off the audio filter so that even the lowest frequencies are received due to decoking?
Best regards.
Joe

**Reply by AlexandreRouma** [reply]
there are no filters, you probably have wrong demodulator setting, never had any issue decoding that kind of signals.

**Comment by om3bc** [comment]
Hi Alexandre!
I did new tests. img1 shows the low frequency spectrum from SDRshapr and
img2 from SDR++. In both cases, an AIRSPY Mini receiver with FM modulation
and a bandwidth of 12500 Hz was used. He made another attempt to receive
the BLUEWALKER-3 satellite. I used plutoSDR with SDR++ and AIRSPY Mini with
SDRSharp. Result: 14 decoded packets with plutoSDR and SDR++, and 94
packets with AIRSPY Mini and SDRsharp. The same antenna, signal divided
into two receivers.
It is a great pity, because your program is very good and easy to use. I
was very happy when I found it. Mainly because I can use my Pluto+ in it.
Pluto can only be used in old versions of SDRsharp, and even there it does
not work correctly. Newer versions are downright awful, complicated and
impractical.
Your program is great. It would be necessary to improve the reception of
digital data a little, and then your program would have no competitor.
Good luck working with the program. I look forward to the newer versions.

Joe


AlexandreRouma ***@***.***> ezt írta (időpont: 2022. szept.
27., K, 11:24):

**Reply by AlexandreRouma** [reply]
1) you didnt post any image
2) you're comparing two different SDR devices.... what did you expect???????
3) 100% user error. SDR++ has no issues with anything digital. Dont assume the software is broken by default.

**Comment by rogerxxxx** [comment]
Some noise filters comparable to SDRSharp/SDR#, would be nice.

Using a recently purchased AirSpy HF+, I find the audio quality (eg. SNR) while listening to shortwave using SDRPlus/SDR+ to be barely usable or barely audibly understandable.  Signals have to be extremely strong for the audio to be understandable above the noise.

I fret digital (or DRM) would almost be impossible with SDRSharp/SDR##'s current noise level.

I spend most of my time within Linux, versus Windows, primarily reserving Windows for testing rather than work.
